### PR TITLE
markup for one cart created;

### PR DESCRIPTION
**Pull request by Kengaroo**
Version desktop

### CODE REVIEW

**Comment by Kengaroo** [comment]
Single cart design (desktop)

**Review by Feals — CHANGES_REQUESTED** [review]
In _cart.html.twig 
line 4 write src="assets/images/icons/Edit.svg" no src="assets/images/icons/edit.svg"
line 7 write src="assets/images/icons/Remove.png" no src="assets/images/icons/remove.png"
line 14 change class name to something like "cart-date-city" and change CSS in function
line 15 delete tag span

**Review by Feals — CHANGES_REQUESTED** [review]
category.html.twig 
delete block header if not we will have a empty header

**Review by Feals — CHANGES_REQUESTED** [review]
ask to merge on dev branch, not main

**Review by AlexMouth — APPROVED** [review]

File: src/Controller/AnnouncementController.php [hunk 1]
```diff
@@ -58,4 +61,26 @@ public function index(): string
         'events' => $this->events, 'active' => $active, 'regions' => $regions, 'selected' => $selected,
         'numpages' => $numpages, 'where' => $where, 'page' => $page]);
     }
+
+    /**
+     * List announcements
+     */
+    public function selectCard(int $id): string
```
Comment (Benja-san) [inline src/Controller/AnnouncementController.php]: better call this method "show" for coherence

File: src/View/Announcement/card.html.twig [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{% extends 'layout.html.twig' %}
```
Comment (Benja-san) [inline src/View/Announcement/card.html.twig]: could you recall this file "detail" ?

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Review by Benja-san — APPROVED** [review]

**Review by Feals — APPROVED** [review]

**Pull request merged**